utils/mixins.py

```diff
@@ -53,19 +53,18 @@ def get_context_data(self, **kwargs: Any) -> dict:
             year = int(self.kwargs["year"])
             month = int(self.kwargs["month"])
             current_date = timezone.now().date()
+            from_date = timezone.now().date().replace(year=year, month=month, day=1)
```

Comment:
Please change it to: `timezone.datetime(year=year, month=month, day=1).date()`

Reply:
Done.